kind.sh

```diff
@@ -141,16 +141,24 @@ parse_command_line() {
 
 install_kind() {
     echo 'Installing kind...'
-    curl -sSLo kind "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
-    chmod +x kind
-    sudo mv kind /usr/local/bin/kind
+    if [ "$RUNNER_OS" == "macOS" ]; then
+        brew install kind
```

Comment:
I think this change the behavior of the action as it would install whatever kind version is defined in brew regardless of the kind version that is set in the action.

Reply:
Good point, let me change that 👍